src/pages/SignIn.tsx

```diff
@@ -33,6 +35,18 @@ export default function SignIn(): ReactElement {
             key: VITE_FORT_MATIC_KEY, // required
           },
         },
+        portis: {
+          package: Portis, // required
+          options: {
+            id: "a180917a-e8d2-43a9-a782-456579bf39b9", // required
+          },
+        },
+        walletconnect: {
+          package: WalletConnectProvider,
+          options: {
+            infuraId: "8cf3cad623da43f9a84ab5ac94230cf6",
```

Comment:
@unicorn-talent  - similar question here for infura.  Imagine it's a working client id?  If so, should be in a config file.

Reply:
yes, those should be in env file